back-end/src/main/java/com/exadel/team3/backend/controllers/TestRestController.java

```diff
@@ -0,0 +1,43 @@
+package com.exadel.team3.backend.controllers;
+
+
+import com.exadel.team3.backend.entities.Model;
+import com.exadel.team3.backend.services.ModelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/app")
+public class TestRestController {
+
+    @Autowired
+    private ModelService modelService;
+
+    @RequestMapping("/models/{name}")
+    public Model getModel(@PathVariable String name){
+        return modelService.getModelByName(name);
+    }
+
+    @RequestMapping("/models")
+    public List<Model> getAllModels(){
+     return modelService.getAllModels();
+    }
+
+    @RequestMapping(value = "/models",method = RequestMethod.POST)
```

Comment:
Use @PostMapping/RestMapping instead of @RequestMapping if API method supports only one type